templates/content-job_listing.php

```diff
@@ -18,7 +18,7 @@
 
 global $post;
 ?>
-<li <?php job_listing_class(); ?> data-longitude="<?php echo esc_attr( $post->geolocation_long ); ?>" data-latitude="<?php echo esc_attr( $post->geolocation_lat ); ?>">
+<li <?php job_listing_class(); ?> data-longitude="<?php echo esc_attr( $post->geolocation_long ); ?>" data-latitude="<?php echo esc_attr( $post->geolocation_lat ); ?>" data-id="<?php echo esc_attr( $post->ID ); ?>">
```

Comment:
`job_listing_class()` outputs a `post-{ID}` class — if we use that instead of `data-id`, then this stat will work for current themes and customized templates. Otherwise folks will have update their templates to include the new attribute, which could take a while and result in some reports of 'stats not working'